service/history/configs/config.go

```diff
@@ -395,6 +396,7 @@ func NewConfig(dc *dynamicconfig.Collection, numberOfShards int32, isAdvancedVis
 
 		NumArchiveSystemWorkflows: dc.GetIntProperty(dynamicconfig.NumArchiveSystemWorkflows, 1000),
 		ArchiveRequestRPS:         dc.GetIntProperty(dynamicconfig.ArchiveRequestRPS, 300), // should be much smaller than frontend RPS
+		ArchiveSignalTimeout:      dc.GetDurationProperty(dynamicconfig.ArchiveSignalTimeout, 1*time.Second),
```

Comment:
We should keep the default value at 300ms. The reason we uses such a short duration is because the call blocks the transfer queue. BTW, we are going to reimplement archival pretty soon. It will be reliable and no need to do this signaling.

Reply:
Makes sense, I set it back. Thanks for the update, looking forward to the new version!